app/controllers/v1/ssj/dashboard_controller.rb

```diff
@@ -0,0 +1,16 @@
+class V1::Ssj::DashboardController < ApiController
+
+  def assigned_steps
+    workflow = Workflow::Instance::Workflow.find_by!(external_identifier: params[:workflow_id])
+    process_ids = Workflow::Instance::Process.where(workflow_id: workflow.id).pluck(:id)
+    steps = Workflow::Instance::Step.where(process_id: process_ids, completed: false).where.not(assignee_id: nil).
+      includes(:process, :documents, :selected_option, :assignee, definition: [:documents]).
+      group_by{|step| step.assignee.external_identifier }
+
+    options = {}
+    options[:include] = ['documents', 'assignee']
+    render json: V1::Ssj::AssignedStepsSerializer.new(steps, options)
+  end
```

Comment:
I just realized by looking at the response, the assignee is serialized multiple times...  that's a lot of data overhead.  not sure how to achieve this but the assignee "include" should only be on the top level object... not each step if that makes sense...

Reply:
Also looking at the person serializer, we should find a way to pass in an option that's like `params[:basic] = true` or something like that that just has what Taylor needs to put a name, email and icon.

Reply:
Maybe for these one off SSJ controllers that arent following JSON API specs anyway, we just construct our own json without going through the serializer - it doesn't seem like it is helping that much anyway... more of an obstacle.  We can still use them to serialize objects but don't need to build the whole response with the assigned steps serializer?  Just a thought.

Reply:
yup this makes sense to me. i agree that we should strip out unneeded data. There isn't a way to pass params to a child serializer via fast json api serializer. so yeah, will try to look into using our own.